Controller/ExceptionController.php

```diff
@@ -126,7 +126,7 @@ protected function getAndCleanOutputBuffering()
      * @param FlattenException $exception
      * @param array            $exceptionMap
      *
-     * @return int|false
+     * @return mixed False, or the $exceptionMap corresponding value
```

Comment:
No, the ``$exceptionMap`` can also contain boolean.
https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/DependencyInjection/Configuration.php#L327
https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/Controller/ExceptionController.php#L161